Client/src/index.css

```diff
@@ -64,6 +65,7 @@
   --env-var-spacing-2: 24px;
   --env-var-spacing-3: 32px;
   --env-var-spacing-4: 40px;
+  --env-var-spacing-5: 65px;
```

Comment:
I think it would be best to name this something like `nav-bar-height`, then we can use it to set both the height of the `NavBar` by targeting `.MuiToolbar-root` and the height of the `SideBar`:

```
(100vh - var(--nav-bar-height));
```

I think that's a net improvement :+1:  It's easier to maintain and the it is very clear from the calculation that the height of the `SideBar` is dependant upon the height of the `NavBar`

Reply:
No need to target `.MuiToolbar-root`
Overflow is fixed already!